nixos/tests/all-tests.nix

```diff
@@ -206,6 +206,7 @@ in {
   bittorrent = handleTest ./bittorrent.nix {};
   blockbook-frontend = handleTest ./blockbook-frontend.nix {};
   blocky = handleTest ./blocky.nix {};
+  bookstack = handleTest ./bookstack.nix {};
```

Comment:
handleTest was deprecated. You should use runTest https://github.com/NixOS/nixpkgs/issues/386873